Tests/Teams.Business.Tests/ManageSprintsServiceTest.cs

```diff
@@ -140,6 +140,50 @@ public async Task AddSprintsAsync_ManageSpiritsServiceReturns_False()
             Assert.IsFalse(result3);
         }
 
+        [Test]
+        public async Task AddSprintsAsync_ManageSpiritsServiceNameCheckReturns_False()
+        {
+            //Arrange
+            var sprint1 = new Sprint { Id = 1, TeamId = 1, Name = "Spr  int1", DaysInSprint = 14, StoryPointInHours = 4, Status = PossibleStatuses.CompletedStatus };
```

Comment:
why 2 spaces?

Reply:
I made it impossible to put several spaces in a row and displayed it here

Reply:
alright then